packages/flutter_adaptive_scaffold/example/pubspec.yaml

```diff
@@ -12,12 +12,14 @@ dependencies:
     sdk: flutter
   flutter_adaptive_scaffold:
     path: ..
-  flutter_lints: ^2.0.0
+  flutter_lints: ^3.0.1
```

Comment:
Why are you updating this? It doesn't appear to be used (which is normal for this repo), and if that's the case it should be removed, not updated.

Reply:
Ok. I'll remove them